src/core/sys/linux/sys/eventfd.d

```diff
@@ -0,0 +1,79 @@
+/**
+ * D header file for GNU/Linux.
+ *
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:   Nemanja Boric
+ */
+module core.sys.linux.sys.eventfd;
+
+version (linux):
+extern (C):
+@system:
+nothrow:
+
+/* Return file descriptor for generic event channel.  Set initial
+   value to count.  */
+int eventfd (uint count, int flags);
+
+/* Read event counter and possibly wait for events.  */
+int eventfd_read (int fd, uint* value);
```

Comment:
Where did you get those signatures from?
According to https://sourceware.org/git/?p=glibc.git;a=blob;f=sysdeps/unix/sysv/linux/sys/eventfd.h;hb=HEAD they should be:
```C
/* Type for event counter.  */
typedef uint64_t eventfd_t;

/* Return file descriptor for generic event channel.  Set initial
   value to COUNT.  */
extern int eventfd (unsigned int __count, int __flags) __THROW;

/* Read event counter and possibly wait for events.  */
extern int eventfd_read (int __fd, eventfd_t *__value);

/* Increment event counter.  */
extern int eventfd_write (int __fd, eventfd_t __value);
```
I suggest you add and use the `eventfd_t` typedef accordingly.